cli.js

```diff
@@ -476,21 +476,40 @@ program
 program
     .command('sandbox:realm:update')
     .description('Update realm settings')
-    .option('-r, --realm <realm>','Realm to update')
-    .option('-m, --max-sandbox-ttl <maxSandboxTTL>','Maximum number of hours a sandbox can live in the realm')
-    .option('-d, --default-sandbox-ttl <defaultSandboxTTL>','Number of hours a sandbox lives in the realm by default')
-    .option('-j, --json','Formats the output in json')
-    .action(function(options) {
-        var realm = ( options.realm ? options.realm : null );
+    .option('-r, --realm <realm>', 'Realm to update')
+    .option('-m, --max-sandbox-ttl <maxSandboxTTL>', 'Maximum number of hours a sandbox can live in the realm')
+    .option('-d, --default-sandbox-ttl <defaultSandboxTTL>', 'Number of hours a sandbox lives in the realm by default')
+    .option('-st, --start-scheduler <startScheduler>', 'Start schedule for all sandboxes under this realm')
```

Comment:
Did you test if the shortcuts `-st` and `-sp` are working? Not sure about that.

Reply:
Removed the shortcuts for all schedule commands